libV2/server/API/internal/prometheusMetrics.js

```diff
@@ -0,0 +1,15 @@
+const { collectDefaultMetrics, register } = require('prom-client');
+
+collectDefaultMetrics({
+    timeout: 10000,
+    gcDurationBuckets: [0.001, 0.01, 0.1, 1, 2, 5],
+    prefix: 'Utapi_v2_',
```

Comment:
The prefix looks fine, just might be more consistent with other prometheus metric names if all lowercase

Reply:
Now that I think about it, the default metrics may be exported without a separate prefix. Indeed Prometheus allows to fetch metrics by label, and those metrics could (should) be labeled by service, hence they could be queried targeting utapi only by providing the proper label selector (say, `{service="utapi"}`). This allows querying those metrics more generally as well.

Reply:
I'm suggesting this since the metrics are applicable to any node.js process, not just utapi. For utapi-specific metrics, e.g. if it was the number of accounts monitored, it would make sense to prefix them.